README.md

```diff
@@ -129,37 +129,72 @@ class YourRepository extends Repository {
     foreignKeys: [{ customerId: String }]
     ```
 
-    The field names will te converted to a database names using conventions. Ex: `customer_id`
+    The field names will converted to a database names using conventions. Ex: `customer_id`
 
 - `knex` - Knex initialize instance
     
     Check Knex [documentation](http://knexjs.org/#Installation-client)
 
 ## Retrieving and Persisting Data
 
-### `findByID`
-Find entities by IDs
+### `find`
```

Comment:
It should contains the doc for each option and how to use

Reply:
Done!